.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+version: 1
+updates:
+  - package-ecosystem: gitsubmodule
+    directory: /
+    schedule:
+      interval: weekly
+    labels:
+      - 'merge-strategy: rebase'
+      - 'type: dependencies'
```

Comment:
FYI: `merge-strategy: rebase` and `type: dependencies` are just labels. Not sure if you want to use them.

I have additional automation that uses these two labels, generates changelog/release notes, and automatically merges PRs.